test/library/draft/DistributedMap/use-distributed-map.chpl

```diff
@@ -81,7 +81,7 @@ proc createReference() {
 
```

Comment:
It looks like at least one of the other draft DistributedMap implementations will need their `items` and references to `these` removed as well